CHANGELOG.md

```diff
@@ -5,7 +5,9 @@ The sections should follow the order `Packaging`, `Added`, `Changed`, `Fixed` an
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## 0.13.0-dev
+## 0.14.0-dev
+
+## 0.13.0
```

Comment:
I think we always used `rc` here? Don't remember.

Reply:
> On master, the version is bumped to `0.3.0-dev` and the `-dev` suffix is removed from the `0.2.0-dev` changelog

Should be fine according to our guideline.